src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/ImageTests/LoadFromBytes.xaml.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.IO;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Uno.UI.Samples.Controls;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media.Imaging;
+
+namespace Uno.UI.Samples.UITests.ImageTestsControl
+{
+	[SampleControlInfo("Image", "LoadFromBytes")]
+	public sealed partial class LoadFromBytes : UserControl
+	{
+		private readonly string _imageUrl = "https://www.spriters-resource.com/resources/sheet_icons/30/32521.png";
+
+		public LoadFromBytes()
+		{
+			InitializeComponent();
+			Loaded += OnLoaded;
+		}
+
+		private async void OnLoaded(object sender, RoutedEventArgs e)
+		{
+			var client = new HttpClient(new CorsByPassHandler());
+
+			var result = await client.GetAsync(_imageUrl);
+			var stream = await result.Content.ReadAsStreamAsync();
```

Comment:
Intead of doing this, add an embedded image (https://github.com/unoplatform/uno/tree/867d997852b05202530221494b68cdd41c40281c/src/SamplesApp/UITests.Shared/Assets) set its build action to EmbeddedREsource and load it like this : https://github.com/unoplatform/uno/blob/867d997852b05202530221494b68cdd41c40281c/src/Uno.UI.Tests/Windows_UI_Xaml_Markup/XamlReaderTests/Given_XamlReader.cs#L742